util/EvobotUtil.js

```diff
@@ -26,3 +26,4 @@ exports.MAX_PLAYLIST_SIZE = config ? config.MAX_PLAYLIST_SIZE : process.env.MAX_
 exports.PRUNING = config ? config.PRUNING : process.env.PRUNING;
 exports.STAY_TIME = config ? config.STAY_TIME : process.env.STAY_TIME;
 exports.DEFAULT_VOLUME = config ? config.DEFAULT_VOLUME: process.env.DEFAULT_VOLUME;
+exports.DJ_ROLE = config ? config.DJ_ROLE : process.env.ROLE 
```

Comment:
We should change `process.env.ROLE` to `process.env.DJ_ROLE` so this config isn't confusing for our Docker users